packages/flutter/lib/src/rendering/table_border.dart

```diff
@@ -197,8 +197,8 @@ class TableBorder {
   void paint(
     Canvas canvas,
     Rect rect, {
-    @required Iterable<double> rows,
-    @required Iterable<double> columns,
+    required Iterable<double> rows,
+    required Iterable<double?> columns,
```

Comment:
Making this nullable is .. surprising? Why's that?

Reply:
haven't gotten to table_border yet